projects/aws/image-builder/builder/builder.go

```diff
@@ -201,7 +208,20 @@ func (b *BuildOptions) BuildImage() {
 			log.Fatalf("Error writing nutanix config file to packer: %v", err)
 		}
 
-		buildCommand := fmt.Sprintf("make -C %s local-build-nutanix-ubuntu-%s", imageBuilderProjectPath, b.OsVersion)
+		var buildCommand string
+		switch b.Os {
+		case Ubuntu:
+			buildCommand = fmt.Sprintf("make -C %s local-build-nutanix-ubuntu-%s", imageBuilderProjectPath, b.OsVersion)
+		case RedHat:
+			buildCommand = fmt.Sprintf("make -C %s local-build-nutanix-redhat-%s", imageBuilderProjectPath, b.OsVersion)
+			commandEnvVars = append(commandEnvVars,
+				fmt.Sprintf("%s=%s", rhelUsernameEnvVar, b.NutanixConfig.RhelUsername),
+				fmt.Sprintf("%s=%s", rhelPasswordEnvVar, b.NutanixConfig.RhelPassword),
+				fmt.Sprintf("%s=%s", rhelImageUrlNutanixEnvVar, b.NutanixConfig.ImageUrl),
+				fmt.Sprintf("%s=%s", "PACKER_NUTANIX_TEST_CONF_FILES", " "),
```

Comment:
im thinking maybe something like this instead? 

https://github.com/aws/eks-anywhere-build-tooling/blob/main/projects/aws/image-builder/builder/builder.go#L173C65-L173C89

thats what we do for cloudstack and baremetal. cc @abhay-krishna

Reply:
@jaxesn I've changed it in similar way like cloudstack and baremetal.